pkg/test-infra/tidb/recommend.go

```diff
@@ -80,6 +80,7 @@ func buildImage(name string) string {
 // RecommendedTiDBCluster does a recommendation, tidb-operator do not have same defaults yet
 func RecommendedTiDBCluster(ns, name string) *Recommendation {
 	enablePVReclaim, exposeStatus := true, true
+	always := corev1.PullAlways
```

Comment:
I am not sure about this one. @fewdan Is this for testing? IMO if not present is better here.